docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md

```diff
@@ -136,3 +136,24 @@ Consider the following example:
     ```output
     You are connected to database "exampledb" as user "adam" on host "localhost" at port "5433".
     ```
+
+## Troubleshooting
+
+Laboratory machines sometimes lack an appropriate intermediate certificate in order to trust the LDAP server certificate.  You can prepend the environment variable `LDAPTLS_REQCERT=never` to test connectivity with ldapsearch:
+
+```sh
+LDAPTLS_REQCERT=never ldapsearch -x -H ldaps://ldapserver.example.org -b dc=example,dc=org 'uid=adam' -D "cn=admin,dc=example,dc=org" -w adminpassword
+```
+
+There are two cases where explicit intermediate CA configuration is needed:
+
+* ldapsearch works correctly with `LDAPTLS_REQCERT=never` but fails otherwise
+* ldapsearch works correctly, but database authentication still fails with a Postgres error message like `LDAP diagnostics: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed`
+
+In either case, you will need to define the intermediate CA in $HOME/ldaprc (or $HOME/.ldaprc if you prefer) for the yugabyte user.  The following example file /home/yugabyte/ldaprc shows the `TLS_CACERT` option pointing to the CA certificate used by the LDAP server.  This CA file will need to be obtained and placed locally on the client machine.
+
+```output
+TLS_CACERT /etc/ssl/certs/ca-bundle.trust.crt
+```
+
+Note the `TLS_CACERT` option *must* be in $HOME/ldaprc -- it will not work in the system-wide openldap configuration file (/etc/openldap/ldap.conf).
```

Comment:
Should be:

If the `TLS_CACERT` option is not set in the `$HOME/ldaprc` directory, it will not work in the system-wide OpenLDAP configuration file `/etc/openldap/ldap.conf`.